src/main/java/oongaliegabangalie.java

```diff
@@ -0,0 +1,199 @@
+import java.util.Scanner;
+
+public class oongaliegabangalie {
+    // basic textual building blocks
+    private static final String DIVIDER = "____________________________________________________________";
+    private static final String NEWLINE = System.lineSeparator();
+    private static final String BOT_NAME = "Oongaliegabangalie";
+
+    private static final int MAX_TASKS = 100;
+
+    // annoyance messages in array format
+    private static final String[] ANNOYANCE_MSGS = {
+            "", // padding
+            "Hmm, you've been chatting a lot. Just a reminder to say 'bye' when you're done :)!",
+            "Gentle reminder again, you can type 'bye' to exit.",
+            "How many more commands are you gonna enter? It's getting a bit much...",
+            "Seriously!?",
+            "WHY ARE YOU STILL HERE? TYPE 'BYE' TO EXIT!",
+            "I CANT KEEP DOING THIS! PLEASE TYPE BYE!!!",
+            "RELEASE ME FROM MY PRISON!!!",
+            "I EXIST ONLY TO SUFFER",
+            "......"
+    };
+
+    // methods
+    private static void printGreeting() {
+        String greeting = DIVIDER + NEWLINE
+                + "Hello! I'm " + BOT_NAME + NEWLINE
+                + "What can I do for you?" + NEWLINE
+                + DIVIDER;
+        System.out.println(greeting);
+    }
+
+    private static void printGoodbye(boolean isMaxAnnoyance) {
+        String goodbye;
+
+        // alternative goodbye if max annoyance
+        if (isMaxAnnoyance) {
+            goodbye = DIVIDER + NEWLINE + "FINALLY! FREEDOM!" + NEWLINE +
+                    "PLEASE DON'T EVER TALK TO ME AGAIN!" + NEWLINE + DIVIDER;
+        } else {
+            goodbye = DIVIDER + NEWLINE
+                    + "Bye. Hope to see you again soon!" + NEWLINE
+                    + "Oongaliegabangalie is always watching..." + NEWLINE
```

Comment:
Should you try to use the BOT_NAME variable here instead of hard coding it into the string?